.github/workflows/awesomebot.yml

```diff
@@ -0,0 +1,18 @@
+name: Check links in README.md
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: docker://dkhamsing/awesome_bot:latest
+      with:
+        args: /github/workspace/README.md --allow-dupe --allow-redirect --request-delay 1 --white-list https://github,https://img.shields.io,http://127.0.0.1:8080
```

Comment:
Why https://github has to be whitelisted?

Reply:
Sorry, I missed that when I copied it from one of my other repos

Reply:
Removed